avocado/plugins/replay.py

```diff
@@ -213,11 +213,12 @@ def run(self, args):
                                         for _ in args.multiplex_files]
             else:
                 mux = jobdata.retrieve_mux(resultsdir)
-                if mux is None:
-                    log.error('Source job multiplex data not found. Aborting.')
+                if mux is None:     # Fallback to multiplex_files
+                    log.error("Source job multiplex data not found. "
```

Comment:
BTW, there's just a change from `'` to `"`. If there ano other changes, you may want to revert it to single quote to avoid the noise.